nucypher/blockchain/eth/sol/source/contracts/Issuer.sol

```diff
@@ -100,7 +100,7 @@ contract Issuer is Upgradeable {
     /**
     * @notice Initialize reserved tokens for reward
     */
-    function initialize(uint256 _reservedReward) public onlyOwner {
+    function initialize(uint256 _reservedReward) external onlyOwner {
```

Comment:
This is a compiler change?

Reply:
just small restriction (can be called only outside of contract), as a result give small gas savings